Azure_Pipelines/deploy_all_services.yaml

```
@@ -0,0 +1,11 @@
+name: Deploy_All
```

yamllint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)